pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -131,33 +131,54 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 }
 
 func setProductLabels(cr *api.KieApp, env *api.Environment) {
-	setObjectLabels(cr, &env.Console, "business-central")
+	envConstants, _ := constants.EnvironmentConstants[cr.Status.Applied.Environment] // is this the only way to read the Image name/s ?
+	setObjectLabels(cr, &env.Console, "business-central", constants.BcImageName, getSubComponentTypeByImageName(constants.BcImageName))
```

Comment:
Not sure this is solution because image name can also have an rhdm prefix ... or maybe the user is leveraging a custom image with a different name. I think we need to be more dynamic in how we determine this.

Reply:
We could consider "application" every image name different to SmartRouter and Controller, is it expected to use metering with custom images ?